### ISSUE TITLE
Question: OpenBoard vs Florisboard

### ISSUE THREAD

**Issue by Offerel** [dslul/openboard#355]
Currently I evaluate again OpenBoard and Florisboard. Personally I like OpenBoard more, but Florisboard has some nice features, like Clipboard view for example.

What I didn't completely understand is the huge size difference of the package. Florisboard is around 7 MB and OpenBoard around 45. Is this only the size of the dictionaries included in the main package? Are there other main differences?

**Comment by dslul** [comment]
You're correct, the size is for the dictionaries. I'm trying to find a solution to reduce the overall size without using internet access, such as "installing" dictionaries from an archive, or alternatively downloading and importing them manually.

**Comment by Offerel** [comment]
AHhh ok. I understand. I don't know if this is possible, but maybe it is possible to include only English (us or uk) in a "base" apk and all other languages in his own apk. Or maybe a apk for Europa, another one for Asia and so on.

**Comment by vojta-horanek** [comment]
What about packing up a compressed version of the dictionaries and decompressing them whenever needed (when the user turns them on, decompress them to the apps private storage, eg. /data/data/org.dslul.openboard...

**Comment by vojta-horanek** [comment]
Tested the compression:
| Decompressed | bzip2    | gzip     | xz       |
|--------------|----------|----------|----------|
| 34708 KB     | 13012 KB | 16104 KB | 12948 KB |
(xz is the best performing, but very slow)

**Comment by NoahAndrews** [comment]
Even gzip is a really nice reduction.

@Offerel This issue should probably be renamed to something like "Reduce APK size of OpenBoard".

**Comment by Offerel** [comment]
This was more a question regarding the real differences in the application (not only the package size), but of course, i can change the subject.

**Comment by dslul** [comment]
> What about packing up a compressed version of the dictionaries and decompressing them whenever needed (when the user turns them on, decompress them to the apps private storage, eg. /data/data/org.dslul.openboard...

that's exactly what I meant with "installing from an archive" :)

> This was more a question regarding the real differences in the application (not only the package size), but of course, i can change the subject.

if you want to discuss such things you can join the matrix channel, so we can keep the issues more focused. I've never tried florisboard, but looking at the feature set developed in such a small time, it seems to be quite a remarkable project.
